src/test/unit/os/EnvTest.cpp

```diff
@@ -43,14 +40,14 @@ void EnvTest::RunTests()
   // get value that is not set
   value = "DONTCHANGEME";
   result = Env::get("NOTDEFINED", value);
-  TESTEQUAL2("get not set result", false, result);
-  TESTEQUAL2_STR("get not set value", "DONTCHANGEME", value.c_str());
+  ASSERT_FALSE(result);
+  EXPECT_STREQ("DONTCHANGEME", value.c_str());
```

Comment:
@scottpurdy `TESTEQUAL2_STR` is defined as `EXPECT_STREQ` in [Tester.hpp](https://github.com/numenta/nupic.core/blob/master/src/nupic/test/Tester.hpp#L158). However, in some cases I've changed EXPECTs to ASSERTs. I don't think it's critical which one is used. As I understand it, both types lead to test failure, but ASSERT stops test execution immediately, whereas EXPECT continues to execute the test and it only fails at the end. So you can have multiple EXPECT failures but only one ASSERT failure in a given test. From what I've read, it's considered best practice to use EXPECT except in cases where moving past that point would lead to an unrecoverable error like a segfault.